src/content/pages/governance/index.md

```diff
@@ -99,7 +99,7 @@ guildsInfo:
           image: /images/162_2.jpg
       joinUrl: https://thresholdnetwork.notion.site/Integrations-Guild-4ff2e5fdea4442d7af19d27342ab8225
 migrationInfo:
-  title: Do you own KEEP or Nu and want to participate in Threshold Goverance?
+  title: Do you own KEEP or NU and want to participate in Threshold Goverance?
```

Comment:
typo: Gover**n**ance

Reply:
Fixed in 1db32882c5f9cd96fbe557333516d0189bc7b918